src/App.jsx

```diff
@@ -1,44 +1,49 @@
-import React from 'react';
+// import React from 'react';
+import { useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+function getContent(data, title) {
+  const currentTab = data.filter(item => item.title === title);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  return currentTab[0].content;
```

Comment:
You're currently using the `title` of the tab to determine the selected tab. This could lead to issues if two tabs have the same title. It's better to use a unique identifier, like the `id`, to determine the selected tab. Please refer to the checklist item 2 for more details.